### PR TITLE
Notifications freelance (Unit Tested)

### PR DESCRIPTION
**Pull request by rafael-longeville**

### CODE REVIEW

**Review by sophiewright31 — APPROVED** [review]
It seems so nice! would love an explanation about it when you can :)

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -106,87 +120,160 @@ function Navbar() {
...
+                            borderBottom: "4px solid #f8c708",
+                          }}
+                        >
+                          <Box bgColor="yalink.green">
+                            <Notifications size="25px" color="#fff" />
+                          </Box>
+                        </MenuButton>
+                      </Flex>
+                      <Box
+                      // h={notifications.length === 0 ? "100px" : "380px"}
```
Comment (juicygoose) [inline src/components/Navbar/Navbar.js:163]: can it be removed ?

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -106,87 +120,160 @@ function Navbar() {
...
+                          p={5}
+                          w="500px"
+                          h="500px"
+                          overflow="auto"
+                        >
+                          <Notification />
+                        </MenuList>
+                      </Box>
+
+                      {/* <Flex alignItems="center">
```
Comment (juicygoose) [inline src/components/Navbar/Navbar.js:176]: can this commented code be removed?

File: src/components/Notifications/Notifications.js [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+                        w="40px"
+                        h="40px"
+                        pt="1"
+                      >
+                        <Box align="center" justify="center">
+                          <DocumentSearch size="25px" color="#fff" />
+                        </Box>
+                      </Box>
+                    )}
+                    {notif.type === "Nouvel e-mail" && (
```
Comment (juicygoose) [inline src/components/Notifications/Notifications.js:159]: You have the opportunity to factorize your code here to define only one function that take one parameter, the icon you want to display !
Have a look at https://github.com/Yalink-Tech/callisto-client/blob/main/src/pages/FreelanceProfile/FreelanceProfile.js#L493 where I did the same !

**Comment by rafael-longeville** [comment]
Fixed all the conversations and responsive issues :)

**Review by juicygoose — APPROVED** [review]

**Pull request merged**